piptools/utils.py

```diff
@@ -702,14 +713,24 @@ def parse_config_file(
             hint=f"Could not parse '{config_file !s}': {value_err !s}",
         )
 
-    # In a TOML file, we expect the config to be under `[tool.pip-tools]`
+    # In a TOML file, we expect the config to be under `[tool.pip-tools]`,
+    # `[tool.pip-compile]` or `[tool.pip-sync]`
```

Comment:
Traditionally, everything concerning a project, should be a part of the same namespaces. We can use subsections for this:

```suggestion
    # `[tool.pip-tools.pip-compile]` or `[tool.pip-tools.pip-sync]`
```

Reply:
Though it's pretty tedious for the user to type this..

Reply:
"Namespaces are one honking great idea". Wouldn't it be more appropriate to use `[tool.pip-tools.compile]`, similar to `python -m piptools.compile`?

Reply:
@theryanwalker fair. That's a better suggestion.

```suggestion
    # `[tool.pip-tools.compile]` or `[tool.pip-tools.sync]`
```

Reply:
@chrysle looks like you forgot to apply the suggested change to the comment line, so I'm reopening this.